functions.py

```diff
@@ -21,6 +21,22 @@ def introStuff(fileName, categoryName):
         w = open('new_files/{}_pleco.txt'.format(fileName), 'w')
         w.write("//{}\n".format(categoryName))
 
+def ankiTime(werdz, file_name,catName):
```

Comment:
I know you were following my lead on the camelCase function naming, but that's a javascript habit i brouth over, and python convention is to use underscores, so change this to `anki_time`. it won't break anything to keep it as is, but it's best to stick to convention

Reply:
i'm also going to change all the other ones to underscore